src/components/metrics/card/updated.js

```diff
@@ -1,5 +1,6 @@
 export function hasMetricUpdated(oldProps, newProps) {
   return (
+    _.get(newProps,'canvasState.metricCardView') === 'analysis' ||
```

Comment:
I feel like we could refine this pretty quickly to be more performant. Actually, could this just be: `oldProps.selectedMetric !== newProps.selectedMetric`?